README.md

```diff
@@ -48,11 +48,11 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
 
 [Kubernetes Code of Conduct]: code-of-conduct.md
 [community page]: https://kubernetes.io/community/
-[contributor guide]: https://sigs.k8s.io/hydrophone/blob/main/CONTRIBUTING.md
+[contributor guide]: https://github.com/kubernetes-sigs/hydrophone/blob/main/CONTRIBUTING.md
 [issue tracker]: https://github.com/kubernetes-sigs/hydrophone/issues
 [@dims]: https://github.com/dims
 [@rjsadow]: https://github.com/rjsadow
-[filing an issue]: https://sigs.k8s.io/hydrophone/issues/new
+[filing an issue]: https://github.com/kubernetes-sigs/hydrophone/issues/new
```

Comment:
```suggestion
[Filing an issue]: https://github.com/kubernetes-sigs/hydrophone/issues/new
```

@UtkarshUmre Can you also maybe adapt this change here and where this link is being called? This would help us to get #124 merged.

Reply:
ya sure